tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -826,6 +827,258 @@ TEST_F( EnergyPlusFixture, EMSManager_CheckIfAnyEMS_OutEMS ) {
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 20 ).Operator, FuncHgAirFnWTdb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 20 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 21 ).Operator, FuncTdpFnTdbTwbPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 21 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 22 ).Operator, FuncTdpFnWPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 22 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 23 ).Operator, FuncHFnTdbW );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 23 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 24 ).Operator, FuncHFnTdbRhPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 24 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 25 ).Operator, FuncTdbFnHW );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 25 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 26 ).Operator, FuncRhovFnTdbRh );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 26 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 27 ).Operator, FuncRhovFnTdbRhLBnd0C ); // fails before #5284, returns FuncRhovFnTdbRh( 41 )
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 27 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 28 ).Operator, FuncRhovFnTdbWPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 28 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 29 ).Operator, FuncRhFnTdbRhov );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 29 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 30 ).Operator, FuncRhFnTdbRhovLBnd0C ); // fails before #5284, returns int const FuncRhFnTdbRhov( 44 )
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 30 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 31 ).Operator, FuncRhFnTdbWPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 31 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 32 ).Operator, FuncTwbFnTdbWPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 32 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 33 ).Operator, FuncVFnTdbWPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 33 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 34 ).Operator, FuncWFnTdpPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 34 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 35 ).Operator, FuncWFnTdbH );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 35 ).NumOperands, 2 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 36 ).Operator, FuncWFnTdbTwbPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 36 ).NumOperands, 3 );
+
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 37 ).Operator, FuncWFnTdbRhPb );
+		EXPECT_EQ( DataRuntimeLanguage::ErlExpression( 37 ).NumOperands, 4 ); // why is this 4?
```

Comment:
W Fn Tdb Rh Pb, or w as a function of 3 independent variables. 4th argument is CalledFrom string for warning messages